pages/price-feeds/troubleshoot/svm.mdx

```diff
@@ -0,0 +1,18 @@
+# Troubleshoot Solana Price Feeds Contract
+
+This reference page is designed to help you troubleshoot common issues you may encounter when using Pyth Price Feeds on SVM chains.
+Follow the steps provided below to diagnose and resolve the issue.
+
+### compilation error[E0277]: the trait bound PriceUpdateV2: anchor_lang::AccountDeserialize is not satisfied
```

Comment:
I feel like this needs more context i.e. this errors happens when you're building a program with [pyth-solana-receiver-sdk](https://crates.io/crates/pyth-solana-receiver-sdk)